DatadogRUM/Sources/RUMMonitor/Scopes/RUMUserActionScope.swift

```diff
@@ -197,6 +202,12 @@ internal class RUMUserActionScope: RUMScope, RUMContextProvider {
         if let event = dependencies.eventBuilder.build(from: actionEvent) {
             writer.write(value: event)
             onActionEventSent(event)
+            interactionToNextViewMetric.trackAction(
+                startTime: actionStartTime,
+                endTime: completionTime,
+                actionType: actionType,
+                in: self.context.activeViewID.orNull
```

Comment:
question/ do we want to track the action when the `activeViewID` is nil?

Reply:
Good catch 👍, the `orNull` was a shorthand I added and it is only confusing in this context. Practically, we should never reach this code if `activeViewID` is `nil` (the action scope must exist within a view scope and that always has an ID). This can be written more explicitly using `if let ...`. Done ✅ 👍.